internal/controlplane/controlplane.go

```diff
@@ -102,6 +109,13 @@ func (d *dashboardClient) Delete(ctx context.Context, obj client.Object) error {
 				return err
 			}
 		}
+
+		ssls, _ := cluster.SSL().List(ctx)
```

Comment:
This operation will completely delete all SSL objects, rather than deleting the objects managed by the current gateway.

Due to the separation mode management, the controller cannot fully manage the lifecycle of the gateway.
It should be noted that when synchronizing DP data in full, only the resources managed by the current controller should be synchronized, otherwise resources may be mistakenly deleted.

Reply:
fixed